modules/programs/tex-fmt.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs, config, lib, ... }:
+
+let
+
+  inherit (lib) mkEnableOption mkPackageOption mkOption literalExpression;
+
+  configDir = if pkgs.stdenv.isDarwin then
```

Comment:
Have you tested darwin with xdg enabled? Does it still write to `Library/Application Support` ?

Reply:
Sorry, was reading above and it seems like you were talking about it that they might just ignore XDG on macos. If that's not the case, it can be fixed in a follow up.